automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeStatusTransitionsSpec.scala

```diff
@@ -47,6 +50,11 @@ class RuntimeStatusTransitionsSpec extends BillingProjectFixtureSpec with Parall
       monitorCreateRuntime(billingProject, runtimeName, runtimeRequest)
       Leonardo.cluster.getRuntime(billingProject, runtimeName).status shouldBe ClusterStatus.Running
 
+      // verify we can hit the proxy when the runtime is running
```

Comment:
This was another improvement. Before, all tests that created a runtime hit the jupyter server API to confirm 'when leo says the app is running I can hit the jupyter server via the proxy', essentially validating leo's `isProxyAvailable` liveliness check. This be done only once, and this seemed like the best place to put it.